src/index.ts

```diff
@@ -3,10 +3,12 @@ import path from 'path'
 import FormData = require('formdata')
 import camelCaseKeys from 'camelcase-keys'
 
-const ClientVersion = require('../package.json').version
```

Comment:
Why get rid of this? Just curious...

Reply:
not part of Marqeta reqest per their documentation